client/src/popups/ConfirmAssignModal.tsx

```diff
@@ -65,105 +69,140 @@ export function ConfirmAssignModal({
+            </Heading>
+          </ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            <Heading size="sm">Review your assignment settings</Heading>
+            {fetcher.data ? (
+              <EditAssignmentSettings
+                maxAttempts={fetcher.data.maxAttempts}
+                individualizeByStudent={fetcher.data.individualizeByStudent}
+                mode={fetcher.data.mode}
+                includeMode={contentType !== "singleDoc"}
+              />
+            ) : (
+              <p>Loading...</p>
+            )}
+
+            <Heading mt="2rem" size="sm">
+              How long would you like this assignment to remain open?
+            </Heading>
+
+            <RadioGroup onChange={setDuration} value={duration} mt="1rem">
+              <Stack direction="column">
+                <Radio value={JSON.stringify({ hours: 48 })}>48 Hours</Radio>
+                <Radio value={JSON.stringify({ weeks: 2 })}>2 Weeks</Radio>
+                <Radio value={JSON.stringify({ years: 1 })}>1 Year</Radio>
+                <Wrap>
+                  <Radio value={"custom"}>Custom</Radio>
+                  <Box>
+                    <FormControl isDisabled={duration !== "custom"}>
+                      <FormLabel hidden={true}>Custom close time</FormLabel>
+                      <Input
+                        zIndex="overlay"
+                        type="datetime-local"
+                        size="sm"
+                        step="60"
+                        width="220px"
+                        value={
+                          customCloseAt
+                            ? customCloseAt
+                            : DateTime.fromSeconds(
+                                Math.round(DateTime.now().toSeconds() / 60) *
+                                  60,
+                              )
+                                .plus({ weeks: 2 })
+                                .toISO({
+                                  includeOffset: false,
+                                  suppressSeconds: true,
+                                  suppressMilliseconds: true,
+                                })
+                        }
+                        onChange={(e) => {
+                          setCustomCloseAt(e.target.value);
+                        }}
+                      />
+                    </FormControl>
+                  </Box>
+                </Wrap>
+              </Stack>
+            </RadioGroup>
+          </ModalBody>
+          <ModalFooter>
+            <Button colorScheme="blue" mr={3} onClick={onClose}>
+              Close
+            </Button>
+            <Button
+              colorScheme="blue"
+              onClick={() => {
+                moveCopyContentOnOpen();
+              }}
+            >
+              Create
+            </Button>
+          </ModalFooter>
+        </ModalContent>
+      </Modal>
+      <MoveCopyContent
```

Comment:
When I click "Create", it's confusing that the next popup says "Copy to Folder". I'm not sure what I'm copying to folder or what is happening.

At the very least, we further customize the title of the popup and the button text.